In [1]:
import pandas as pd

In [2]:
data = pd.read_csv("C:\\Users\\bhawna\\Downloads\\Shark Tank India Dataset.csv")

In [3]:
data.head(10)

episode_number  pitch_number           brand_name  \
0               1             1  BluePine Industries   
1               1             2        Booz scooters   
2               1             3  Heart up my Sleeves   
3               2             4           Tagz Foods   
4               2             5       Head and Heart   
5               2             6         Agro tourism   
6               3             7          Qzense Labs   
7               3             8            Peeschute   
8               3             9                 NOCD   
9               4            10                Cosiq   

                                            idea  deal  pitcher_ask_amount  \
0                                   Frozen Momos     1                50.0   
1  Renting e-bike for mobility in private spaces     1                40.0   
2                             Detachable Sleeves     1                25.0   
3                           Healthy Potato Chips     1                70.0   
4                       Brain Development Course     0                50.0   
5                                        Tourism     0                50.0   
6                        Food Freshness Detector     0               100.0   
7                           Disposable Urine Bag     1                75.0   
8                                   Energy Drink     1                50.0   
9                           Intelligent Skincare     1                50.0   

   ask_equity  ask_valuation  deal_amount  deal_equity  ...  ashneer_deal  \
0        5.00        1000.00         75.0        16.00  ...             1   
1       15.00         266.67         40.0        50.00  ...             1   
2       10.00         250.00         25.0        30.00  ...             0   
3        1.00        7000.00         70.0         2.75  ...             1   
4        5.00        1000.00          0.0         0.00  ...             0   
5        5.00        1000.00          0.0         0.00  ...             0   
6        0.25       40000.00          0.0         0.00  ...             0   
7        4.00        1875.00         75.0         6.00  ...             0   
8        2.00        2500.00         20.0        15.00  ...             0   
9        7.50         666.67         50.0        25.00  ...             0   

   anupam_deal  aman_deal  namita_deal  vineeta_deal  peyush_deal  \
0            0          1            0             1            0   
1            0          0            0             1            0   
2            1          0            0             1            0   
3            0          0            0             0            0   
4            0          0            0             0            0   
5            0          0            0             0            0   
6            0          0            0             0            0   
7            0          1            0             0            0   
8            0          0            0             1            0   
9            1          0            0             1            0   

   ghazal_deal  total_sharks_invested  amount_per_shark  equity_per_shark  
0            0                      3              25.0          5.333333  
1            0                      2              20.0         25.000000  
2            0                      2              12.5         15.000000  
3            0                      1              70.0          2.750000  
4            0                      0               0.0          0.000000  
5            0                      0               0.0          0.000000  
6            0                      0               0.0          0.000000  
7            0                      1              75.0          6.000000  
8            0                      1              20.0         15.000000  
9            0                      2              25.0         12.500000  

[10 rows x 28 columns]

In [4]:
import numpy as np

In [5]:
data.iloc[np.where((data["ashneer_deal"]==1) & (data["amount_per_shark"]) & (data["equity_per_shark"]))][:]

episode_number  pitch_number             brand_name  \
0                 1             1    BluePine Industries   
1                 1             2          Booz scooters   
3                 2             4             Tagz Foods   
15                6            16            Skippi Pops   
18                7            19     Raising Superstars   
21                8            22           Beyond Snack   
23                8            24          Motion Breeze   
29               10            30              EventBeep   
38               13            39        The Yarn Bazaar   
45               16            46           Bamboo India   
49               17            50  Find Your Kicks India   
50               17            51          Aas Vidyalaya   
55               18            56                   Otua   
58               19            59                WeSTOCK   
63               20            64               IN A CAN   
64               21            65             Get a Whey   
67               22            68         Hair Originals   
108              33           109             Tweek Labs   
109              33           110                 Proxgy   
110              34           111     Nomad Food Project   
114              35           115          Jain Shikanji   

                                              idea  deal  pitcher_ask_amount  \
0                                     Frozen Momos     1                50.0   
1    Renting e-bike for mobility in private spaces     1                40.0   
3                             Healthy Potato Chips     1                70.0   
15                                        Ice-Pops     1                45.0   
18                           Child Development App     0               100.0   
21                             Kerala Banana Chips     1                50.0   
23                       Smart Electric Motorcycle     1                30.0   
29                           Student Community App     1                30.0   
38                                Yarn-Trading App     1                50.0   
45                                 Bamboo Products     1                80.0   
49                                  Sneaker Resale     1                50.0   
50                                      EdTech App     1               150.0   
55                           Electric Auto Vehicle     1               100.0   
58                  Livestock health monitoring AI     1                50.0   
63                                   Can Cocktails     1                50.0   
64                             Sugar-Free Icecream     1               100.0   
67                         Natural Hair Extensions     1                60.0   
108                                     Sportswear     1                40.0   
109                                             VR     1                35.0   
110                                     Bacon Jams     1                40.0   
114                                       Lemonade     1                40.0   

     ask_equity  ask_valuation  deal_amount  deal_equity  ...  ashneer_deal  \
0           5.0        1000.00         75.0        16.00  ...             1   
1          15.0         266.67         40.0        50.00  ...             1   
3           1.0        7000.00         70.0         2.75  ...             1   
15          5.0         900.00        100.0        15.00  ...             1   
18          2.0        5000.00        100.0         4.00  ...             1   
21          2.5        2000.00         50.0         2.50  ...             1   
23          3.0        1000.00         30.0         6.00  ...             1   
29          2.0        1500.00         30.0         3.00  ...             1   
38          2.0        2500.00        100.0        10.00  ...             1   
45          4.0        2000.00         50.0         3.50  ...             1   
49         10.0         500.00         50.0        25.

# Ashneer invested in 21 different brands


In [6]:
data.groupby(["ashneer_deal"])["amount_per_shark"].sum()

ashneer_deal
0    1627.166936
1     494.333333
Name: amount_per_shark, dtype: float64

# * Ashneer invested 4 Cr 94 lakhs in Total

In [7]:
data.groupby(["ashneer_deal"])["amount_per_shark"].max()

ashneer_deal
0    100.0
1     70.0
Name: amount_per_shark, dtype: float64

In [8]:
data.groupby(["ashneer_deal"])["equity_per_shark"].max()

ashneer_deal
0    75.0
1    25.0
Name: equity_per_shark, dtype: float64

In [9]:
data.iloc[np.where((data["ashneer_deal"]==1) & (data["equity_per_shark"]==25.0))]

episode_number  pitch_number     brand_name  \
1               1             2  Booz scooters   

                                            idea  deal  pitcher_ask_amount  \
1  Renting e-bike for mobility in private spaces     1                40.0   

   ask_equity  ask_valuation  deal_amount  deal_equity  ...  ashneer_deal  \
1        15.0         266.67         40.0         50.0  ...             1   

   anupam_deal  aman_deal  namita_deal  vineeta_deal  peyush_deal  \
1            0          0            0             1            0   

   ghazal_deal  total_sharks_invested  amount_per_shark  equity_per_shark  
1            0                      2              20.0              25.0  

[1 rows x 28 columns]

# Ashneer has maximum equity of 25% of Booz scooters with an amount of 20 lakhs invested

# Ashneer invested 70 lakhs as the maximum in comparison to all his investments  with 2.75 equity in Tagz Foods
 

In [10]:
f_data = pd.DataFrame(columns = (["Total no. of investments"],["max._Equity"],["max.Equity company name"],["Total_Investment"],["max. Investment"]),index = ("Ashneer Grover","Anupam","Aman","Namita","vineeta","peyush","ghazal"))

In [11]:
data.iloc[np.where(data["anupam_deal"]==1)].shape

(24, 28)

# ANUMPAM invested in 24 companies in total

In [12]:
data.groupby(["anupam_deal"])["amount_per_shark"].sum()

anupam_deal
0    1587.666667
1     533.833603
Name: amount_per_shark, dtype: float64

# Anupam invested 5 Cr 33 lakhs Total

In [13]:
data.groupby(["anupam_deal"])["amount_per_shark"].max()

anupam_deal
0    100.0
1     50.0
Name: amount_per_shark, dtype: float64

In [14]:
data.iloc[np.where((data["anupam_deal"]==1) & (data["amount_per_shark"]==50.0))]

episode_number  pitch_number                brand_name           idea  \
12               5            13               Revamp Moto         E-Bike   
31              11            32  ARRCOAT Surface Textures  Wall Building   
78              25            79                 PawsIndia   Dog Products   

    deal  pitcher_ask_amount  ask_equity  ask_valuation  deal_amount  \
12     1               100.0         1.0        10000.0        100.0   
31     1                50.0         5.0         1000.0         50.0   
78     1                50.0         4.0         1250.0         50.0   

    deal_equity  ...  ashneer_deal  anupam_deal  aman_deal  namita_deal  \
12          1.5  ...             0            1          1            0   
31         15.0  ...             0            1          0            0   
78         15.0  ...             0            1          0            0   

    vineeta_deal  peyush_deal  ghazal_deal  total_sharks_invested  \
12             0            0            0                      2   
31             0            0            0                      1   
78             0            0            0                      1   

    amount_per_shark  equity_per_shark  
12              50.0              0.75  
31              50.0             15.00  
78              50.0             15.00  

[3 rows x 28 columns]

In [15]:
data.groupby(["anupam_deal"])["equity_per_shark"].max()

anupam_deal
0    75.0
1    17.5
Name: equity_per_shark, dtype: float64

In [16]:
data.iloc[np.where((data["anupam_deal"]==1) & (data["equity_per_shark"]==17.5))]

episode_number  pitch_number    brand_name   idea  deal  \
75              24            76  The Sass Bar  Gifts     1   

    pitcher_ask_amount  ask_equity  ask_valuation  deal_amount  deal_equity  \
75                40.0         8.0          500.0         50.0         35.0   

    ...  ashneer_deal  anupam_deal  aman_deal  namita_deal  vineeta_deal  \
75  ...             0            1          0            0             0   

    peyush_deal  ghazal_deal  total_sharks_invested  amount_per_shark  \
75            0            1                      2              25.0   

    equity_per_shark  
75              17.5  

[1 rows x 28 columns]

# anupam has max equity of 17.5 % of The sass Bar with an amount of 25 lakhs invested

# anupam's max amount invested was 50 lakhs in 3 different companies

In [17]:
data.iloc[np.where(data["aman_deal"]==1)].shape

(28, 28)

In [18]:
data.groupby(["aman_deal"])["amount_per_shark"].sum()

aman_deal
0    1234.000252
1     887.500017
Name: amount_per_shark, dtype: float64

In [19]:
data.groupby(["aman_deal"])["equity_per_shark"].max()

aman_deal
0    75.0
1    40.0
Name: equity_per_shark, dtype: float64

In [20]:
data.iloc[np.where((data["aman_deal"]==1) & (data["equity_per_shark"]==40.0))]

episode_number  pitch_number        brand_name                  idea  \
42              15            43  Hammer Lifestyle  Smart Audio Products   

    deal  pitcher_ask_amount  ask_equity  ask_valuation  deal_amount  \
42     1                30.0         3.0         1000.0        100.0   

    deal_equity  ...  ashneer_deal  anupam_deal  aman_deal  namita_deal  \
42         40.0  ...             0            0          1            0   

    vineeta_deal  peyush_deal  ghazal_deal  total_sharks_invested  \
42             0            0            0                      1   

    amount_per_shark  equity_per_shark  
42             100.0              40.0  

[1 rows x 28 columns]

In [21]:
data.groupby(["aman_deal"])["amount_per_shark"].max()

aman_deal
0    100.0
1    100.0
Name: amount_per_shark, dtype: float64

# Aman has invested in a total of 28 brands with a total amount of 8 Cr 87 lakhs as of now and he holds a max of 40%equity in the company Hammer Lifestyle with max investment of 1 Cr.

In [22]:
data.iloc[np.where(data["namita_deal"]==1)].shape

(22, 28)

In [23]:
data.groupby(["namita_deal"])["amount_per_shark"].sum()

namita_deal
0    1473.166667
1     648.333603
Name: amount_per_shark, dtype: float64

In [24]:
data.groupby(["namita_deal"])["equity_per_shark"].max()

namita_deal
0    75.0
1    25.0
Name: equity_per_shark, dtype: float64

In [25]:
data.iloc[np.where((data["namita_deal"]==1) & (data["equity_per_shark"]==25.0))]

episode_number  pitch_number     brand_name     idea  deal  \
106              33           107  Colour Me Mad  Insoles     1   

     pitcher_ask_amount  ask_equity  ask_valuation  deal_amount  deal_equity  \
106                40.0        10.0          400.0         40.0         25.0   

     ...  ashneer_deal  anupam_deal  aman_deal  namita_deal  vineeta_deal  \
106  ...             0            0          0            1             0   

     peyush_deal  ghazal_deal  total_sharks_invested  amount_per_shark  \
106            0            0                      1              40.0   

     equity_per_shark  
106              25.0  

[1 rows x 28 columns]

In [26]:
data.groupby(["namita_deal"])["amount_per_shark"].max()

namita_deal
0    100.0
1     75.0
Name: amount_per_shark, dtype: float64

In [27]:
data.iloc[np.where((data["namita_deal"]==1) & (data["amount_per_shark"]==75.0))]

episode_number  pitch_number      brand_name                idea  deal  \
33              12            34  Auli Lifestyle  Ayurvedic Products     1   

    pitcher_ask_amount  ask_equity  ask_valuation  deal_amount  deal_equity  \
33                75.0         4.0         1875.0         75.0         15.0   

    ...  ashneer_deal  anupam_deal  aman_deal  namita_deal  vineeta_deal  \
33  ...             0            0          0            1             0   

    peyush_deal  ghazal_deal  total_sharks_invested  amount_per_shark  \
33            0            0                      1              75.0   

    equity_per_shark  
33              15.0  

[1 rows x 28 columns]

# Namita has invested in a total of 22 Brands with a total amount of 6 Cr and 48 lakhs invested till now. She holds a max of 25% equity in the company Colour Me Bad and her max investment reached 75 Lakhs for a company called Auli lifestyle.

In [28]:
data.iloc[np.where(data["vineeta_deal"]==1)].shape

(15, 28)

In [29]:
data.groupby(["vineeta_deal"])["amount_per_shark"].sum()

vineeta_deal
0    1793.166936
1     328.333333
Name: amount_per_shark, dtype: float64

In [30]:
data.groupby(["vineeta_deal"])["equity_per_shark"].max()

vineeta_deal
0    75.0
1    25.0
Name: equity_per_shark, dtype: float64

In [31]:
data.iloc[np.where((data["vineeta_deal"]==1) & (data["equity_per_shark"]==25.0))]

episode_number  pitch_number     brand_name  \
1               1             2  Booz scooters   

                                            idea  deal  pitcher_ask_amount  \
1  Renting e-bike for mobility in private spaces     1                40.0   

   ask_equity  ask_valuation  deal_amount  deal_equity  ...  ashneer_deal  \
1        15.0         266.67         40.0         50.0  ...             1   

   anupam_deal  aman_deal  namita_deal  vineeta_deal  peyush_deal  \
1            0          0            0             1            0   

   ghazal_deal  total_sharks_invested  amount_per_shark  equity_per_shark  
1            0                      2              20.0              25.0  

[1 rows x 28 columns]

In [32]:
data.groupby(["vineeta_deal"])["amount_per_shark"].max()

vineeta_deal
0    100.0
1     40.0
Name: amount_per_shark, dtype: float64

In [33]:
data.iloc[np.where((data["vineeta_deal"]==1) & (data["amount_per_shark"]==40.0))]

episode_number  pitch_number    brand_name             idea  deal  \
95              30            96  Kabaddi Adda  All-Kabaddi App     1   

    pitcher_ask_amount  ask_equity  ask_valuation  deal_amount  deal_equity  \
95                80.0         1.0         8000.0         80.0          6.0   

    ...  ashneer_deal  anupam_deal  aman_deal  namita_deal  vineeta_deal  \
95  ...             0            0          0            1             1   

    peyush_deal  ghazal_deal  total_sharks_invested  amount_per_shark  \
95            0            0                      2              40.0   

    equity_per_shark  
95               3.0  

[1 rows x 28 columns]

# Vineeta has invested in a total of 15 Brands with a total amount of 3 Cr and 28 lakhs invested till now. She holds a max of 25% equity in the company Booz Scooters and her max investment reached 40 Lakhs for a company called Kabaddi adda.

In [34]:
data.iloc[np.where(data["peyush_deal"]==1)].shape

(27, 28)

In [35]:
data.groupby(["peyush_deal"])["amount_per_shark"].sum()

peyush_deal
0    1401.833350
1     719.666919
Name: amount_per_shark, dtype: float64

In [36]:
data.groupby(["peyush_deal"])["equity_per_shark"].max()

peyush_deal
0    40.0
1    75.0
Name: equity_per_shark, dtype: float64

In [37]:
data.iloc[np.where((data["peyush_deal"]==1) & (data["equity_per_shark"]==75.0))]

episode_number  pitch_number     brand_name        idea  deal  \
65              21            66  Sid07 Designs  Inventions     1   

    pitcher_ask_amount  ask_equity  ask_valuation  deal_amount  deal_equity  \
65                47.0        10.0          470.0         25.0         75.0   

    ...  ashneer_deal  anupam_deal  aman_deal  namita_deal  vineeta_deal  \
65  ...             0            0          0            0             0   

    peyush_deal  ghazal_deal  total_sharks_invested  amount_per_shark  \
65            1            0                      1              25.0   

    equity_per_shark  
65              75.0  

[1 rows x 28 columns]

In [38]:
data.groupby(["peyush_deal"])["amount_per_shark"].max()

peyush_deal
0    100.0
1    100.0
Name: amount_per_shark, dtype: float64

In [40]:
data.iloc[np.where((data["peyush_deal"]==1) & (data["amount_per_shark"]==100))]

episode_number  pitch_number          brand_name                 idea  \
87              27            88  Insurance Samadhan  Insurance Solutions   

    deal  pitcher_ask_amount  ask_equity  ask_valuation  deal_amount  \
87     1               100.0         1.0        10000.0        100.0   

    deal_equity  ...  ashneer_deal  anupam_deal  aman_deal  namita_deal  \
87          4.0  ...             0            0          0            0   

    vineeta_deal  peyush_deal  ghazal_deal  total_sharks_invested  \
87             0            1            0                      1   

    amount_per_shark  equity_per_shark  
87             100.0               4.0  

[1 rows x 28 columns]

# Peyush has invested in a total of 27 Brands with a total amount of 7 Cr and 19 lakhs invested till now. He holds a max of 75% equity in the company Sid Designs and his max investment reached 1 Cr for a company called Insurance Samadhan.

In [41]:
data.iloc[np.where(data["ghazal_deal"]==1)].shape

(7, 28)

In [42]:
data.groupby(["ghazal_deal"])["amount_per_shark"].sum()

ghazal_deal
0    1991.500017
1     130.000253
Name: amount_per_shark, dtype: float64

In [43]:
data.groupby(["ghazal_deal"])["equity_per_shark"].max()

ghazal_deal
0    75.0
1    17.5
Name: equity_per_shark, dtype: float64

In [44]:
data.iloc[np.where((data["ghazal_deal"]==1) & (data["equity_per_shark"]==17.5))]

episode_number  pitch_number    brand_name   idea  deal  \
75              24            76  The Sass Bar  Gifts     1   

    pitcher_ask_amount  ask_equity  ask_valuation  deal_amount  deal_equity  \
75                40.0         8.0          500.0         50.0         35.0   

    ...  ashneer_deal  anupam_deal  aman_deal  namita_deal  vineeta_deal  \
75  ...             0            1          0            0             0   

    peyush_deal  ghazal_deal  total_sharks_invested  amount_per_shark  \
75            0            1                      2              25.0   

    equity_per_shark  
75              17.5  

[1 rows x 28 columns]

In [45]:
data.groupby(["ghazal_deal"])["amount_per_shark"].max()

ghazal_deal
0    100.000000
1     33.333333
Name: amount_per_shark, dtype: float64

In [46]:
data.iloc[np.where((data["ghazal_deal"]==1))]

episode_number  pitch_number                brand_name  \
75               24            76              The Sass Bar   
79               25            80       Sunfox Technologies   
85               27            86        Watt Technovations   
88               28            89                  Humpy A2   
90               28            91  Gold Safe Solutions Ind.   
91               29            92               Wakao Foods   
110              34           111        Nomad Food Project   

                      idea  deal  pitcher_ask_amount  ask_equity  \
75                   Gifts     1            40.00000         8.0   
79     Portable ECG Device     1           100.00000         2.0   
85     Ventilated PPE Kits     1             0.00101        10.0   
88   Organic Milk Products     1            75.00000         4.0   
90   Anti-Suicidal Fan Rod     1            50.00000         5.0   
91      Jackfruit Products     1            75.00000         5.0   
110             Bacon Jams     1            40.00000        10.0   

     ask_valuation  deal_amount  deal_equity  ...  ashneer_deal  anupam_deal  \
75          500.00     50.00000         35.0  ...             0            1   
79         5000.00    100.00000          6.0  ...             0            1   
85            0.01      0.00101          4.0  ...             0            1   
88         1875.00    100.00000         15.0  ...             0            0   
90         1000.00     50.00000         30.0  ...             0            0   
91         1500.00     75.00000         21.0  ...             0            0   
110         400.00     40.00000         20.0  ...             1            0   

     aman_deal  namita_deal  vineeta_deal  peyush_deal  ghazal_deal  \
75           0            0             0            0            1   
79           0            1             1            1            1   
85           0            1             0            1            1   
88           0            0             1            1            1   
90           0            0             1            1            1   
91           0            1             1            0            1   
110          0            1             1            0            1   

     total_sharks_invested  amount_per_shark  equity_per_shark  
75                       2         25.000000              17.5  
79                       5         20.000000               1.2  
85                       4          0.000253               1.0  
88                       3         33.333333               5.0  
90                       3         16.666667              10.0  
91                       3         25.000000               7.0  
110                      4         10.000000               5.0  

[7 rows x 28 columns]

# Ghazal has invested in a total of 7 Brands with a total amount of 1 Cr and 30 lakhs invested till now. He holds a max of 17.5% equity in the company The Sass Bar and his max investment reached 33.3 lakhs for a company called Humpy A2.

In [47]:
f_data

(Total no. of investments,) (max._Equity,)  \
Ashneer Grover                         NaN            NaN   
Anupam                                 NaN            NaN   
Aman                                   NaN            NaN   
Namita                                 NaN            NaN   
vineeta                                NaN            NaN   
peyush                                 NaN            NaN   
ghazal                                 NaN            NaN   

               (max.Equity company name,) (Total_Investment,)  \
Ashneer Grover                        NaN                 NaN   
Anupam                                NaN                 NaN   
Aman                                  NaN                 NaN   
Namita                                NaN                 NaN   
vineeta                               NaN                 NaN   
peyush                                NaN                 NaN   
ghazal                                NaN                 NaN   

               (max. Investment,)  
Ashneer Grover                NaN  
Anupam                        NaN  
Aman                          NaN  
Namita                        NaN  
vineeta                       NaN  
peyush                        NaN  
ghazal                        NaN

In [48]:
f_data["Total_no. of investments"] = [21,24,28,22,15,27,7]

In [49]:
f_data["max_equity"] = ["25%","17.5%","40%","25%","25%","75%","17.5%"]

In [50]:
f_data["max_quity_company_name"] = ["Booz Scooters","PAWS INDIA","Hammer Lifestyle","Colour Me Bad","Booz Scooters","Sid Designs","The Sass Bar"]

In [51]:
f_data["Total_investment(in lakhs)"] = [494,533,887,648,328,719,130]

In [52]:
f_data["Max_Investment (in lakhs)"] = [70,50,100,75,40,100,33.3]

In [53]:
F_data = f_data.iloc[:,5:]

In [54]:
F_data

Total_no. of investments max_equity max_quity_company_name  \
Ashneer Grover                        21        25%          Booz Scooters   
Anupam                                24      17.5%             PAWS INDIA   
Aman                                  28        40%       Hammer Lifestyle   
Namita                                22        25%          Colour Me Bad   
vineeta                               15        25%          Booz Scooters   
peyush                                27        75%            Sid Designs   
ghazal                                 7      17.5%           The Sass Bar   

                Total_investment(in lakhs)  Max_Investment (in lakhs)  
Ashneer Grover                         494                       70.0  
Anupam                                 533                       50.0  
Aman                                   887                      100.0  
Namita                                 648                       75.0  
vineeta                                328                       40.0  
peyush                                 719                      100.0  
ghazal                                 130                       33.3

# THE DATA ABOVE DEPICTS THE IMPORTANT INFORMATION WHICH COULD BE GATHERED FROM THE WHOLE DATA SET.

In [55]:
data["total_sharks_invested"].max()

5

In [56]:
data.iloc[np.where((data["total_sharks_invested"]==5))]

episode_number  pitch_number             brand_name                 idea  \
15               6            16            Skippi Pops             Ice-Pops   
49              17            50  Find Your Kicks India       Sneaker Resale   
63              20            64               IN A CAN        Can Cocktails   
79              25            80    Sunfox Technologies  Portable ECG Device   

    deal  pitcher_ask_amount  ask_equity  ask_valuation  deal_amount  \
15     1                45.0         5.0          900.0        100.0   
49     1                50.0        10.0          500.0         50.0   
63     1                50.0         2.0         2500.0        100.0   
79     1               100.0         2.0         5000.0        100.0   

    deal_equity  ...  ashneer_deal  anupam_deal  aman_deal  namita_deal  \
15         15.0  ...             1            1          1            1   
49         25.0  ...             1            1          1            1   
63         10.0  ...             1            1          1            1   
79          6.0  ...             0            1          0            1   

    vineeta_deal  peyush_deal  ghazal_deal  total_sharks_invested  \
15             1            0            0                      5   
49             0            1            0                      5   
63             0            1            0                      5   
79             1            1            1                      5   

    amount_per_shark  equity_per_shark  
15              20.0               3.0  
49              10.0               5.0  
63              20.0               2.0  
79              20.0               1.2  

[4 rows x 28 columns]

# THE BRANDS WHICH PULLED THE MAXIMUM NUMBER OF SHARKS WERE THE ABOVE 4.

In [57]:
data["total_sharks_invested"].min()

0

In [59]:
data.iloc[np.where((data["total_sharks_invested"]==0))]

episode_number  pitch_number                 brand_name  \
4                 2             5             Head and Heart   
5                 2             6               Agro tourism   
6                 3             7                Qzense Labs   
10                4            11               JhaJi Achaar   
13                5            14               Hungry Heads   
14                5            15         Shrawani Engineers   
17                6            18                    Hecolll   
19                7            20                   Torch-it   
20                7            21              La Kheer Deli   
26                9            27            Kabira Handmade   
30               11            31                 Gopal's 56   
34               12            35                   SweeDesi   
40               14            41         Morikko Pure Foods   
41               14            42       Good Good Piggy Bank   
46               16            47                Flying Furr   
51               17            52                     Outbox   
53               18            54            Mommy's Kitchen   
54               18            55          India Hemp and Co   
56               18            57                  Anthyesti   
57               19            58                      Ethik   
59               19            60                  KetoIndia   
60               19            61                 Magic lock   
62               20            63                   Bakarmax   
68               22            69             Poo de Cologne   
69               22            70            Moonshine Meads   
70               22            71                    Falhari   
72               23            73               Urban Monkey   
73               23            74             Guardian Gears   
74               23            75                Modern Myth   
77               24            78             Nuskha Kitchen   
80               25            81                     Alpino   
82               26            83           Julaa Automation   
84               27            85               Theka Coffee   
86               27            87        Aliste Technologies   
89               28            90               Kunafa World   
92               29            93                 PDD Falcon   
93               29            94                 PlayBox TV   
94               30            95  Sippline Drinking Shields   
96               30            97           Shades of Spring   
97               30            98                   Scholify   
98               31            99                 Scrapshala   
99               31           100                 Sabjikothi   
101              31           102                     Astrix   
102              32           103               Thea and Sid   
103              32           104            Experential Etc   
105              32           106                   Med Tech   
107              33           108                     Mavi's   
111              34           112                Twee in One   
112              34           113              Green Protein   
113              34           114                    On2Cook   
115              35           116                      Woloo   
116              35           117               Elcare India   

                                                 idea  deal  \
4                            Brain Development Course     0   
5                                             Tourism     0   
6                             Food Freshness Detector     0   
10                                             Pickle     0   
13               Restaurant serving 80 types of Maggi     0   
14                                Belly Button Shaper     0   
17                         Pollution Resistant Fabric     0   
19               Gadgets for visually impaired people     1   
20                        Kheer in variety of

# From the above data it is clear that 52 brands were such that nobody liked investing in them.